source/cgal/intersections.cc

```diff
@@ -225,130 +225,133 @@ namespace CGALWrappers
     // signatures follow the convection that the first entity has an intrinsic
     // dimension higher than the second one.
 
+    template <int spacedim>
```

Comment:
it looks like this is only for spacedim = 2. Do we need the template parameter?

Reply:
I deleted the obsolete template parameter for both cases.